trialexp/process/pyphotometry/utils.py

```diff
@@ -688,11 +688,11 @@ def compute_PCA(
     
     scaler = StandardScaler()
     pca = PCA(0.7, random_state=33)
-    pca.fit(scaler.fit_transform(X.iloc[past_id]))
+    pca.fit(scaler.fit_transform(data))
     
     Xt = pca.inverse_transform(
         pca.transform(
-            scaler.transform(X.iloc[future_id])
+            scaler.transform(data)
```

Comment:
I was referring to this line : data instead of X.iloc[future_id]
 Not sure if that could break anything in your code but I am not sure if you are using PCA at all?